docs/source/queries/result.md

```diff
@@ -51,7 +51,7 @@ if let Some(rows) = session.query("SELECT a, b from ks.tab", &[]).await?.rows {
 # }
 ```
 
-## Parsing using convenience methods
+# Parsing using convenience methods
```

Comment:
It doesn't look like this should be there, the whole chapter is about query result, but now this is a sibling to `Query Result` chapter.
After undoing this, there will probably be some other changes in this file required.